lib/SDIR/Translate/translateToSDFG.cpp

```diff
@@ -242,8 +243,14 @@ LogicalResult translation::translateToSDFG(SDFGNode &op, JsonEmitter &jemit) {
   jemit.endObject(); // in_connectors
 
   jemit.startNamedObject("out_connectors");
-  // TODO: Print out_connectors
-  jemit.printKVPair("__return", "null", /*stringify=*/false);
+  // TODO: Implement multiple return values
+  if (op.getNumResults() == 1) {
+    jemit.printKVPair("__return", "null", /*stringify=*/false);
+  } else if (op.getNumResults() > 1) {
+    emitError(op.getLoc(), "Multiple return values not implemented yet");
```

Comment:
for the future, multiple return values take the form `__return_%d` starting from zero